SwrveSDK/SDK/Talk/SwrveMessageController.h

```diff
@@ -115,7 +115,8 @@ typedef void (^SwrveCustomButtonPressedCallback) (NSString* action);
 @interface SwrveMessageController : NSObject<SwrveMessageDelegate, SwrveMessageEventHandler>
 
 @property (nonatomic) Swrve*  analyticsSDK;                                             /*!< Analytics SDK reference. */
-@property (nonatomic, retain) UIColor* backgroundColor;                                 /*!< Background color of in-app messages. */
+@property (nonatomic, retain) UIColor* inAppMessageBackgroundColor;                     /*!< Background color of in-app messages. */
```

Comment:
This will have to be documented in the upgrade guide. Will break compilation of someone using this property on <= 4.4